modules/swagger-codegen/src/main/resources/Java/api.mustache

```diff
@@ -52,6 +52,9 @@ public class {{classname}} {
    * @return {{returnType}}
    {{/returnType}}
    * @throws ApiException if fails to make API call
+   {{#isDeprecated}}
+   * @Deprecated
```

Comment:
@hisener Isn't this a javadoc comment and not an annotation because the line starts with `*`?